hsc/crawler.py

```diff
@@ -113,26 +148,28 @@ def get_submissions(self, submissions):
 					track_folder_name = track['name'].strip().replace(' ', '')
 					track_url = self.domain_url.format(track['track_slug'], track['slug'])
 					parent_folder_name = track['track_name'].strip().replace(' ', '')
-					folder_name = os.path.join(parent_folder_name ,track_folder_name)
-				
-				if language in self.special_extensions:
-					file_extension = '.' + self.special_extensions[language]
+					folder_name = os.path.join(parent_folder_name, track_folder_name)
```

Comment:
Can we reduce the method complexity by introducing some methods(getters) like `getFolderName(parentFolderName)`?

Reply:
yeah, I agree but we should not do that in this PR.